### ISSUE TITLE
Publishing pointclouds in os_sensor frame is misleading

### ISSUE THREAD

**Issue by peci1** [ouster-lidar/ouster_example#295]
I know the documentation describes the situation with frames quite extensively, but generally people expect that raw data from a sensor are published in the acquisition frame. That would be `os_lidar`. But the example ROS node transforms and publishes the data in `os_sensor`, which is rotated and shifted in Z.

One possible consequence of misinterpreting the frames is usage of the scans in a raytracing pipeline. These pipelines usually take the frame_id of the incoming scan and cast rays from the origin of the given frame. But if the frame is `os_sensor`, the cast rays are invalid because the origin is shifted lower than the real optical origin of the rays.

I thus suggest changing the data frame to `os_lidar`. Users who do not care about the origin of observation should not be affected (except receiving the data in a different frame), and users interested in the observation point would get the expected data by default.

**Comment by dgarbuzov** [comment]
Yep, agreed. Iirc the code predates the frames defined in the manual. Gonna leave this open for the next major version
